gecko/core/metrics.py

```
@@ -106,6 +117,8 @@ def __init__(
         self.description = description
         self.buckets = tuple(sorted(buckets))
 
+        # 存储所有观测值以计算百分位数（注意内存占用）
```

Ruff (0.14.7)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)